docs/development/local_setup.md

```diff
@@ -191,10 +191,10 @@ The commands below will configure your `minikube` with the absolute minimum reso
 
 #### Start minikube
 
-First, start `minikube` with at least Kubernetes v1.11.x. The default CPU and memory settings of the minikube machine are not sufficient to host the control plane of a shoot cluster, thus the minimal resources should be 3 CPUs and 4096 MB memory, while the recommended are 4 CPUs and 8192 MB memory.
+First, start `minikube` with at least Kubernetes v1.11.x. The default CPU and memory settings of the minikube machine are not sufficient to host the control plane of a shoot cluster, thus the minimal resources should be 3 CPUs and 4096 MB memory, while the recommended are 4 CPUs and 8192 MB memory. If you are going to use the `local` provider, then the minikube cluster must be able to create `NodePort` service on port `443`
```

Comment:
What's the rationale for changing the port? This may be very limitative as this port may already be used.